### DISCUSSION TITLE
有什么办法可以简单的进行shader的debug吗？

### DISCUSSION THREAD

**Discussion by iceiceice123** [BoomingTech/Piccolo#169] [Q&A] [answered]
以前只用过renderdoc这种工具，想问下pilot中有什么类似的方法可以debug吗？

**Comment by kwbm** [comment, marked as answer]
修改ShaderCompile.cmake文件里的编译参数，让shader编译结果包含调试信息，然后重新编译工程，就可以使用renderdoc debug了
<img width="732" alt="cmake-shader-debug" src="https://user-images.githubusercontent.com/102910208/166666345-297ab5b3-6bf9-4d12-b69d-7230c80a1f88.png">
